### ISSUE TITLE
2025-02-21, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#90]
# Today's Arxiv Papers

(1) [LServe: Efficient Long-sequence LLM Serving with Unified Sparse Attention](http://arxiv.org/abs/2502.14866v1), 2025-02-20

(2) [Aligning LLMs to Ask Good Questions A Case Study in Clinical Reasoning](http://arxiv.org/abs/2502.14860v1), 2025-02-20

(3) [FR-Spec: Accelerating Large-Vocabulary Language Models via Frequency-Ranked Speculative Sampling](http://arxiv.org/abs/2502.14856v1), 2025-02-20

(4) [Prompt-to-Leaderboard](http://arxiv.org/abs/2502.14855v1), 2025-02-20

(5) [CLIPPER: Compression enables long-context synthetic data generation](http://arxiv.org/abs/2502.14854v1), 2025-02-20

(6) [GATE: Graph-based Adaptive Tool Evolution Across Diverse Tasks](http://arxiv.org/abs/2502.14848v1), 2025-02-20

(7) [Red-Teaming LLM Multi-Agent Systems via Communication Attacks](http://arxiv.org/abs/2502.14847v1), 2025-02-20

(8) [Scaling Text-Rich Image Understanding via Code-Guided Synthetic Multimodal Data Generation](http://arxiv.org/abs/2502.14846v1), 2025-02-20

(9) [Revealing and Mitigating Over-Attention in Knowledge Editing](http://arxiv.org/abs/2502.14838v1), 2025-02-20

(10) [Towards Economical Inference: Enabling DeepSeek's Multi-Head Latent Attention in Any Transformer-based LLMs](http://arxiv.org/abs/2502.14837v1), 2025-02-20

(11) [Middle-Layer Representation Alignment for Cross-Lingual Transfer in Fine-Tuned LLMs](http://arxiv.org/abs/2502.14830v1), 2025-02-20

(12) [Fundamental Limitations in Defending LLM Finetuning APIs](http://arxiv.org/abs/2502.14828v1), 2025-02-20

(13) [A Survey of Model Architectures in Information Retrieval](http://arxiv.org/abs/2502.14822v1), 2025-02-20

(14) [eC-Tab2Text: Aspect-Based Text Generation from e-Commerce Product Tables](http://arxiv.org/abs/2502.14820v1), 2025-02-20

(15) [Dynamic Low-Rank Sparse Adaptation for Large Language Models](http://arxiv.org/abs/2502.14816v1), 2025-02-20

(16) [Optimizing Model Selection for Compound AI Systems](http://arxiv.org/abs/2502.14815v1), 2025-02-20

(17) [From RAG to Memory: Non-Parametric Continual Learning for Large Language Models](http://arxiv.org/abs/2502.14802v1), 2025-02-20

(18) [A Multi-Agent Perspective on Modern Information Retrieval](http://arxiv.org/abs/2502.14796v1), 2025-02-20

(19) [Rapid Word Learning Through Meta In-Context Learning](http://arxiv.org/abs/2502.14791v1), 2025-02-20

(20) [SurveyX: Academic Survey Automation via Large Language Models](http://arxiv.org/abs/2502.14776v1), 2025-02-20

(21) [Determining Layer-wise Sparsity for Large Language Models Through a Theoretical Perspective](http://arxiv.org/abs/2502.14770v1), 2025-02-20

(22) [Logic-RL: Unleashing LLM Reasoning with Rule-Based Reinforcement Learning](http://arxiv.org/abs/2502.14768v1), 2025-02-20

(23) [Tree-of-Debate: Multi-Persona Debate Trees Elicit Critical Thinking for Scientific Comparative Analysis](http://arxiv.org/abs/2502.14767v1), 2025-02-20

(24) [Step-by-Step Fact Verification System for Medical Claims with Explainable Reasoning](http://arxiv.org/abs/2502.14765v1), 2025-02-20

(25) [EquivaMap: Leveraging LLMs for Automatic Equivalence Checking of Optimization Formulations](http://arxiv.org/abs/2502.14760v1), 2025-02-20

(26) [On the Influence of Context Size and Model Choice in Retrieval-Augmented Generation Systems](http://arxiv.org/abs/2502.14759v1), 2025-02-20

(27) [TritonBench: Benchmarking Large Language Model Capabilities for Generating Triton Operators](http://arxiv.org/abs/2502.14752v1), 2025-02-20

(28) [Large Language Models Struggle to Describe the Haystack without Human Help: Human-in-the-loop Evaluation of LLMs](http://arxiv.org/abs/2502.14748v1), 2025-02-20

(29) [Multi-Agent Coordination across Diverse Applications: A Survey](http://arxiv.org/abs/2502.14743v1), 2025-02-20

(30) [SuperGPQA: Scaling LLM Evaluation across 285 Graduate Disciplines](http://arxiv.org/abs/2502.14739v1), 2025-02-20

(31) [EAGER-LLM: Enhancing Large Language Models as Recommenders through Exogenous Behavior-Semantic Integration](http://dx.doi.org/10.1145/3696410.3714933), 2025-02-20

(32) [WavRAG: Audio-Integrated Retrieval Augmented Generation for Spoken Dialogue Models](http://arxiv.org/abs/2502.14727v1), 2025-02-20

(33) [Entity Framing and Role Portrayal in the News](http://arxiv.org/abs/2502.14718v1), 2025-02-20

(34) [From Knowledge Generation to Knowledge Verification: Examining the BioMedical Generative Capabilities of ChatGPT](http://arxiv.org/abs/2502.14714v1), 2025-02-20

(35) [I-MCTS: Enhancing Agentic AutoML via Introspective Monte Carlo Tree Search](http://arxiv.org/abs/2502.14693v1), 2025-02-20

(36) [Bridging the Gap: Transforming Natural Language Questions into SQL Queries via Abstract Query Pattern and Contextual Schema Markup](http://arxiv.org/abs/2502.14682v1), 2025-02-20

(37) [How to Get Your LLM to Generate Challenging Problems for Evaluation](http://arxiv.org/abs/2502.14678v1), 2025-02-20

(38) [Data-Constrained Synthesis of Training Data for De-Identification](http://arxiv.org/abs/2502.14677v1), 2025-02-20

(39) [Explanations of Deep Language Models Explain Language Representations in the Brain](http://arxiv.org/abs/2502.14671v1), 2025-02-20

(40) [AlphaMaze: Enhancing Large Language Models' Spatial Intelligence via GRPO](http://arxiv.org/abs/2502.14669v1), 2025-02-20

(41) [InstructAgent: Building User Controllable Recommender via LLM Agent](http://arxiv.org/abs/2502.14662v1), 2025-02-20

(42) [Beyond the Surface: Uncovering Implicit Locations with LLMs for Personalized Local News](http://arxiv.org/abs/2502.14660v1), 2025-02-20

(43) [Edit Once, Update Everywhere: A Simple Framework for Cross-Lingual Knowledge Synchronization in LLMs](http://arxiv.org/abs/2502.14645v1), 2025-02-20

(44) [LIFT: Improving Long Context Understanding of Large Language Models through Long Input Fine-Tuning](http://arxiv.org/abs/2502.14644v1), 2025-02-20

(45) [Length-Controlled Margin-Based Preference Optimization without Reference Model](http://arxiv.org/abs/2502.14643v1), 2025-02-20

(46) [How Far are LLMs from Being Our Digital Twins? A Benchmark for Persona-Based Behavior Chain Simulation](http://arxiv.org/abs/2502.14642v1), 2025-02-20

(47) [CER: Confidence Enhanced Reasoning in LLMs](http://arxiv.org/abs/2502.14634v1), 2025-02-20

(48) [Augmenting Coaching with GenAI: Insights into Use, Effectiveness, and Future Potential](http://arxiv.org/abs/2502.14632v1), 2025-02-20

(49) [Synergistic Fusion of Multi-Source Knowledge via Evidence Theory for High-Entropy Alloy Discovery](http://arxiv.org/abs/2502.14631v1), 2025-02-20

(50) [PEARL: Towards Permutation-Resilient LLMs](http://arxiv.org/abs/2502.14628v1), 2025-02-20

(51) [Reward Models Identify Consistency, Not Causality](http://arxiv.org/abs/2502.14619v1), 2025-02-20

(52) [Serving Models, Fast and Slow:Optimizing Heterogeneous LLM Inferencing Workloads at Scale](http://arxiv.org/abs/2502.14617v1), 2025-02-20

(53) [FIND: Fine-grained Information Density Guided Adaptive Retrieval-Augmented Generation for Disease Diagnosis](http://arxiv.org/abs/2502.14614v1), 2025-02-20

(54) [Behavioral Analysis of Information Salience in Large Language Models](http://arxiv.org/abs/2502.14613v1), 2025-02-20

(55) ["Don't Forget the Teachers": Towards an Educator-Centered Understanding of Harms from Large Language Models in Education](http://arxiv.org/abs/2502.14592v1), 2025-02-20

(56) [ReVISE: Learning to Refine at Test-Time via Intrinsic Self-Verification](http://arxiv.org/abs/2502.14565v1), 2025-02-20

(57) [Plan-over-Graph: Towards Parallelable LLM Agent Schedule](http://arxiv.org/abs/2502.14563v1), 2025-02-20

(58) [Can LLMs Predict Citation Intent? An Experimental Analysis of In-context Learning and Fine-tuning on Open LLMs](http://arxiv.org/abs/2502.14561v1), 2025-02-20

(59) [Less is More: Improving LLM Alignment via Preference Data Selection](http://arxiv.org/abs/2502.14560v1), 2025-02-20

(60) [LLM-based User Profile Management for Recommender System](http://arxiv.org/abs/2502.14541v1), 2025-02-20

(61) [LoRA-GGPO: Mitigating Double Descent in LoRA Fine-Tuning via Gradient-Guided Perturbation Optimization](http://arxiv.org/abs/2502.14538v1), 2025-02-20

(62) [CORBA: Contagious Recursive Blocking Attacks on Multi-Agent Systems Based on Large Language Models](http://arxiv.org/abs/2502.14529v1), 2025-02-20

(63) [Generative adversarial networks vs large language models: a comparative study on synthetic tabular data generation](http://arxiv.org/abs/2502.14523v1), 2025-02-20

(64) [Can LLMs Simulate L2-English Dialogue? An Information-Theoretic Analysis of L1-Dependent Biases](http://arxiv.org/abs/2502.14507v1), 2025-02-20

(65) [How Much Knowledge Can You Pack into a LoRA Adapter without Harming LLM?](http://arxiv.org/abs/2502.14502v1), 2025-02-20

(66) [MLGym: A New Framework and Benchmark for Advancing AI Research Agents](http://arxiv.org/abs/2502.14499v1), 2025-02-20

(67) [Enhancing Language Multi-Agent Learning with Multi-Agent Credit Re-Assignment for Interactive Environment Generalization](http://arxiv.org/abs/2502.14496v1), 2025-02-20

(68) [StructFlowBench: A Structured Flow Benchmark for Multi-turn Instruction Following](http://arxiv.org/abs/2502.14494v1), 2025-02-20

(69) [NLoRA: Nyström-Initiated Low-Rank Adaptation for Large Language Models](http://arxiv.org/abs/2502.14482v1), 2025-02-20

(70) [Unshackling Context Length: An Efficient Selective Attention Approach through Query-Key Compression](http://arxiv.org/abs/2502.14477v1), 2025-02-20

(71) [Argument-Based Comparative Question Answering Evaluation Benchmark](http://arxiv.org/abs/2502.14476v1), 2025-02-20

(72) [Enhancing Smart Environments with Context-Aware Chatbots using Large Language Models](http://arxiv.org/abs/2502.14469v1), 2025-02-20

(73) [Narrative-Driven Travel Planning: Geoculturally-Grounded Script Generation with Evolutionary Itinerary Optimization](http://arxiv.org/abs/2502.14456v1), 2025-02-20

(74) [Optimal word order for non-causal text generation with Large Language Models: the Spanish case](http://dx.doi.org/10.1016/j.patrec.2025.02.010), 2025-02-20

(75) [LLM4FaaS: No-Code Application Development using LLMs and FaaS](http://arxiv.org/abs/2502.14450v1), 2025-02-20

(76) [PredictaBoard: Benchmarking LLM Score Predictability](http://arxiv.org/abs/2502.14445v1), 2025-02-20

(77) [Natural Language Generation](http://dx.doi.org/10.1007/978-3-031-68582-8), 2025-02-20

(78) [Token-Level Density-Based Uncertainty Quantification Methods for Eliciting Truthfulness of Large Language Models](http://arxiv.org/abs/2502.14427v1), 2025-02-20

(79) [A Survey on Data Contamination for Large Language Models](http://arxiv.org/abs/2502.14425v1), 2025-02-20

(80) [ChatVLA: Unified Multimodal Understanding and Robot Control with Vision-Language-Action Model](http://arxiv.org/abs/2502.14420v1), 2025-02-20

(81) [Towards Efficient Automatic Self-Pruning of Large Language Models](http://arxiv.org/abs/2502.14413v1), 2025-02-20

(82) [Unstructured Evidence Attribution for Long Context Query Focused Summarization](http://arxiv.org/abs/2502.14409v1), 2025-02-20

(83) [HPS: Hard Preference Sampling for Human Preference Alignment](http://arxiv.org/abs/2502.14400v1), 2025-02-20

(84) [Leveraging Small LLMs for Argument Mining in Education: Argument Component Identification, Classification, and Assessment](http://arxiv.org/abs/2502.14389v1), 2025-02-20

(85) [Rumor Detection by Multi-task Suffix Learning based on Time-series Dual Sentiments](http://arxiv.org/abs/2502.14383v1), 2025-02-20

(86) [S*: Test Time Scaling for Code Generation](http://arxiv.org/abs/2502.14382v1), 2025-02-20

(87) [Retrieval-Augmented Process Reward Model for Generalizable Mathematical Reasoning](http://arxiv.org/abs/2502.14361v1), 2025-02-20

(88) [Triangulating LLM Progress through Benchmarks, Games, and Cognitive Tests](http://arxiv.org/abs/2502.14359v1), 2025-02-20

(89) [Self-Improvement Towards Pareto Optimality: Mitigating Preference Conflicts in Multi-Objective Alignment](http://arxiv.org/abs/2502.14354v1), 2025-02-20

(90) [SR-LLM: Rethinking the Structured Representation in Large Language Model](http://arxiv.org/abs/2502.14352v1), 2025-02-20

(91) [FlowAgent: Achieving Compliance and Flexibility for Workflow Agents](http://arxiv.org/abs/2502.14345v1), 2025-02-20

(92) [Earlier Tokens Contribute More: Learning Direct Preference Optimization From Temporal Decay Perspective](http://arxiv.org/abs/2502.14340v1), 2025-02-20

(93) [A Survey on Feedback-based Multi-step Reasoning for Large Language Models on Mathematics](http://arxiv.org/abs/2502.14333v1), 2025-02-20

(94) [SolSearch: An LLM-Driven Framework for Efficient SAT-Solving Code Generation](http://arxiv.org/abs/2502.14328v1), 2025-02-20

(95) [ChemHTS: Hierarchical Tool Stacking for Enhancing Chemical Agents](http://arxiv.org/abs/2502.14327v1), 2025-02-20

(96) [Beyond Self-Talk: A Communication-Centric Survey of LLM-Based Multi-Agent Systems](http://arxiv.org/abs/2502.14321v1), 2025-02-20

(97) [Line Goes Up? Inherent Limitations of Benchmarks for Evaluating Large Language Models](http://arxiv.org/abs/2502.14318v1), 2025-02-20

(98) [ParallelComp: Parallel Long-Context Compressor for Length Extrapolation](http://arxiv.org/abs/2502.14317v1), 2025-02-20

(99) [Unveiling Cultural Blind Spots: Analyzing the Limitations of mLLMs in Procedural Text Comprehension](http://arxiv.org/abs/2502.14315v1), 2025-02-20

(100) [Efficient AI in Practice: Training and Deployment of Efficient LLMs for Industry Applications](http://arxiv.org/abs/2502.14305v1), 2025-02-20
